var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -46,6 +48,7 @@ class PyYt(PythonPackage):
     depends_on("py-ipython", type=('build', 'run'))
     depends_on("py-ipython@:6.99", type=('build', 'run'), when="^python@:2.99")
     depends_on("py-matplotlib", type=('build', 'run'))
+    depends_on("py-matplotlib@:3.2.2", type=('build', 'run'), when="@:3.6.0")
     depends_on("py-numpy", type=('build', 'run'))
     depends_on("py-scipy", type=('build', 'run'), when="+scipy")
     depends_on("py-setuptools", type=('build', 'run'))
```

Comment:
It looks like Python 2.7:2.8,3.5: is now required. You should also add type=build/run. Also, cython 0.24+ is needed. Many other dependencies are also missing versions:
```python
    install_requires=[                                                                   
        'matplotlib>=1.5.3',                                                             
        'setuptools>=19.6',                                                              
        'sympy>=1.0',                                                                    
        'numpy>=1.10.4',                                                                 
        'IPython>=1.0',                                                                  
    ],        
```

Reply:
Thank you, fixed.